app/src/main/res/layout-large-land/fragment_video_detail.xml

```diff
@@ -2,12 +2,12 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/video_item_detail"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="?attr/windowBackground">
 
     <LinearLayout
-        android:id="@+id/video_item_detail"
```

Comment:
Wouldn't there be side-effects by just moving the id like this? I think `tools:viewBindingIgnore="true"` would be safer since this layout doesn't need view binding anyway.

Reply:
I found that a compile error is generated even with `tools:viewBindingIgnore="true"`, and it's needed [here](https://github.com/TeamNewPipe/NewPipe/pull/4814) anyway.

Reply:
Ah okay. If it's required later than it's the better approach. I'm just curious because the root layout also covers `overlay_layout` and there might be unexpected behavior if, for example, the visibility of `video_item_detail`  is changed (but I didn't look whether there is such case xD)